src/sagemaker/inputs.py

```diff
@@ -165,6 +165,63 @@ class TransformInput(object):
     """Create a class containing all the parameters.
 
     It can be used when calling ``sagemaker.transformer.Transformer.transform()``
+
+    Args:
+        data (str): The S3 location of the input data that the model can consume.
+        data_type (str): The data type for an elastic inference accelerator.
```

Comment:
Actually should it be "The data type for a batch transform job"? "an elastic inference accelerator" is only a device.